ci/travis/build-manylinux.sh

```diff
@@ -6,7 +6,11 @@ rm -rf build ssh2/libssh2.*
 python ci/appveyor/fix_version.py .
 
 docker pull $docker_tag || echo
-docker build --cache-from $docker_tag ci/docker/manylinux -t $docker_tag
+if [[ `uname -m` == "aarch64" ]]; then
+  docker build --cache-from $docker_tag ci/docker/manylinux -t $docker_tag -f ci/docker/manylinux/Dockerfile.aarch64
```

Comment:
Can we also add change to docker tag to `${docker_tag}-aarch64` for this build, so we do not overwrite tags.

Reply:
Done.